configure

```diff
@@ -139,17 +142,23 @@ class Configure:
 
 def main():
     parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Maybe this argument should have action of store_true?

Reply:
We want the conda environment to be created by default, so I've changed the command line argument to `--skip_conda_env` and given it an action of store_true -- therefore keeping the default action as creating the cenv.